code/modules/clothing/masks/gasmask.dm

```diff
@@ -206,7 +206,7 @@
 					
 								"halt" 			= "HALT! HALT! HALT! HALT!",
 								"bobby" 		= "Stop in the name of the Law.",
-								"compliance" 	        = "Compliance is in your best ineterest.",
+								"compliance" 	        = "Compliance is in your best interest.",
 								"justice"		= "Prepare for justice!",
 								"running"		= "Running will only increase your sentence",

```

Comment:
just if you're altering this file.... you might as well add a fullstop/period at the end of this sentence, and the `harry` statement.

All others have line/sentence enders.